sway/input/seat.c

```diff
@@ -1430,6 +1432,9 @@ void seat_apply_config(struct sway_seat *seat,
 
 struct seat_config *seat_get_config(struct sway_seat *seat) {
 	struct seat_config *seat_config = NULL;
+	if (!config) {
```

Comment:
Hmm. But `seat_apply_input_config` unconditionally accesses `config`. Does something make `cursor_get_timeout` different?

Reply:
No, I just tested again and have now removed the line and verified it by using some additional logging. I applied the changes for the crashes first and moved the event source at the very last. All that is required is moving the arm of the timer.

Reply:
OK. If removing completely the checks make you uncomfortable, we can add assertions instead.